Helper/Factory/OrderItemFactory.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+namespace SwedbankPay\Checkout\Helper\Factory;
+
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Model\Quote\Item as QuoteItem;
+use Magento\Sales\Api\Data\OrderItemInterface;
+use Magento\Sales\Model\Order;
+use SwedbankPay\Api\Service\Paymentorder\Resource\Collection\Item\OrderItem;
+
+class OrderItemFactory
+{
+    /**
+     * @param QuoteItem $quoteItem
+     * @return OrderItem
+     */
+    public function createByQuoteItem(QuoteItem $quoteItem)
+    {
+        $sku = $quoteItem->getSku();
+        $name = $quoteItem->getName();
+        $type = 'PRODUCT';
+        $itemClass = 'ProductGroup1';
+        $quantity = (float) $quoteItem->getQty();
+        $unitPrice = (int) round($quoteItem->getPriceInclTax() * 100);
+        $amount = (int) round($quoteItem->getRowTotalInclTax() * 100);
+        $vatAmount = (int) round($quoteItem->getTaxAmount() * 100);
+        $vatPercent = (int) round($quoteItem->getTaxPercent() * 100);
+        $discountPrice = (int) round($quoteItem->getDiscountAmount() * 100);
+        $description = $quoteItem->getDescription();
+
+        return $this->create(
+            $sku,
+            $name,
+            $type,
+            $itemClass,
+            $quantity,
+            $unitPrice,
+            $amount,
+            $vatAmount,
+            $vatPercent,
+            $discountPrice,
+            $description
+        );
+    }
+
+    /**
+     * @param OrderItemInterface $orderItem
+     * @return OrderItem
+     */
+    public function createByOrderItem(OrderItemInterface $orderItem)
+    {
+        $sku = $orderItem->getSku();
+        $name = $orderItem->getName();
+        $type = 'PRODUCT';
+        $itemClass = 'ProductGroup1';
```

Comment:
Please don't hard code `$itemClass` to `ProductGroup1`. If there's a "product category" or similar field in Magento you can use, please use that as long as you ensure it adheres to its specification:

>The classification of the order item. Can be used for assigning the order item to a specific product category, such as `MobilePhone`. Note that class cannot contain spaces and must follow the regex pattern `[\w]* (a-zA-Z0-9_)`. Swedbank Pay may use this field for statistics.